botc/commands/info/notes.py

```diff
@@ -136,29 +136,34 @@ async def info_add(self, ctx, *, args: str):
         # Ensure that we have enough values in the list
         args += ['', '']
 
-        target_player = BOTCUtils.get_player_from_string(args[0])
-        if target_player is not None:
-            args.pop(0)
-
         try:
             target_phase = int(args[0])
+            target_player = None
             args.pop(0)
-
-            if target_phase > globvars.master_state.game._chrono.cycle:
-                raise commands.BadArgument()
         except ValueError:
-            target_phase = None
+            target_player = BOTCUtils.get_player_from_string(args[0])
+            if target_player is not None:
+                args.pop(0)
+
+            try:
+                target_phase = int(args[0])
+                args.pop(0)
+
+                if target_phase > globvars.master_state.game._chrono.cycle:
+                    raise commands.BadArgument()
+            except ValueError:
+                target_phase = None
 
         # Remove the two dummy elements
         args.pop()
         args.pop()
 
         rem = ' '.join(args)
-        if rem == '':
+        if rem.strip() == '':
```

Comment:
```suggestion
        if not rem.strip():
```
This is more Pythonic, it's unnecessary to explicitly compare to the empty string.